surelock/src/main/java/com/smashingboxes/surelock/SurelockDefaultDialog.java

```diff
@@ -61,24 +74,70 @@ public void onCreate(Bundle savedInstanceState) {
 
         // Do not create a new Fragment when the Activity is re-created such as orientation changes.
         setRetainInstance(true);
-        setStyle(DialogFragment.STYLE_NO_TITLE, android.R.style.Theme);
+
+        TypedArray attrs = getActivity().obtainStyledAttributes(styleId, R.styleable
+                .SurelockDefaultDialog);
+        int dialogTheme = attrs.getResourceId(R.styleable.SurelockDefaultDialog_sl_dialog_theme,
+                Integer.MAX_VALUE);
```

Comment:
I could be wrong in this, but I think it's possible for resources to be addressed at the integer max value #.
It may be best to just set default to 0 here (and check 0 below).